dist/image-sequencer-ui.js

```diff
@@ -79,6 +79,7 @@ window.onload = function() {
     ui.selectNewStepUi();
     ui.addStepUi();
     $(this).removeClass('selected');
+    // ui.removeStepUi();
```

Comment:
Just one question. What does this comment denote and is it required?

Reply:
My apologies, that was a mistake and thought that it was removed